go/vt/vttablet/onlineddl/executor.go

```diff
@@ -669,37 +670,24 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream) er
 	}
 
 	// Preparation is complete. We proceed to cut-over.
-
-	// lock keyspace:
-	{
-		lctx, unlockKeyspace, err := e.ts.LockKeyspace(ctx, e.keyspace, "OnlineDDLCutOver")
```

Comment:
No more talking to `topo`, no locking of the shard, we do everything internally t othe `tabletserver`